oasst-shared/oasst_shared/schemas/protocol.py

```diff
@@ -405,6 +405,7 @@ def __new__(cls, label: str, widget: LabelWidget, display_text: str = "", help_t
     humor = "humor", LabelWidget.likert, "Humorous content including sarcasm"
     helpfulness = "helpfulness", LabelWidget.likert, "Helpfulness of the message"
     creativity = "creativity", LabelWidget.likert, "Creativity"
+    controversial = "controversial", LabelWidget.likert, "Controversial"
```

Comment:
Why likert? What would be the two extreme ends of the scale? IMO something is either controversial or not, it could be a flag option.

Reply:
I will not agree with this. Using my definition of `controversial`, there are levels of controversy. 

As per discussion at https://github.com/LAION-AI/Open-Assistant/issues/1710 , "controversial" does not mean `true` nor `untrue` nor `undecidable`. It only means that it has a high potential to cause anger, bias, disagreement, drama and heated discussion.